pipeline/tests/pil.rs

```diff
@@ -3,10 +3,13 @@ use powdr_number::Bn254Field;
 use powdr_number::GoldilocksField;
 use powdr_pipeline::test_util::{
     assert_proofs_fail_for_invalid_witnesses, assert_proofs_fail_for_invalid_witnesses_estark,
-    assert_proofs_fail_for_invalid_witnesses_halo2,
-    assert_proofs_fail_for_invalid_witnesses_pilcom, gen_estark_proof, test_halo2,
+    assert_proofs_fail_for_invalid_witnesses_pilcom, gen_estark_proof, test_halo2, test_plonky3,
     verify_test_file,
 };
+
+#[cfg(feature = "halo2")]
```

Comment:
I added it because when running the tests with the `plonky3` feature only, `assert_proofs_fail_for_invalid_witnesses_halo2` is not used, which triggers a warning. Not an absolute requirement, but also makes sense that a halo2 function is only included if the halo2 feature is on.

Reply:
Ok my bad, reverted